shell/platform/embedder/embedder_external_view.cc

```diff
@@ -74,6 +74,7 @@ bool EmbedderExternalView::HasEngineRenderedContents() {
   DlOpSpy dl_op_spy;
   slice_->dispatch(dl_op_spy);
   has_engine_rendered_contents_ = dl_op_spy.did_draw() && !slice_->is_empty();
+  // NOLINTNEXTLINE(bugprone-unchecked-optional-access)
```

Comment:
Hoo boy. This seems like a trivial one it should have been able to deal with...

Reply:
The access patterns are pretty established: https://clang.llvm.org/extra/clang-tidy/checks/bugprone/unchecked-optional-access.html!